### ISSUE TITLE
BBCT Lobby creation crash

### ISSUE THREAD

**Issue by DarkTree01** [RipleyTom/rpcn#90]
### Quick Summary 
---

Latest build 1.3.0 stated it added servers for Arcsys games, though when attempting to make a lobby in BlazBlue: Calamity Trigger, the game crashes (does not close, is frozen and given the following message):

![image](https://github.com/user-attachments/assets/546da0fc-0fde-4f66-a6f3-89b615a22ee8)

This issue also happens when attempting a Ranked matchmaking search, though that takes longer to come to the same result and may require another person matchmaking.

### Details/Reproduction
---

Default settings with FPS limit at 60, and RPCN network settings are unchanged aside from Network Status being set to Connected and PSN status set to RPCN. United States region. Enter the "Network" menu and create a lobby, everything works up to that point but the game will freeze.

### System Information
---
version: RPCS3 ver. 0.033-16941-eb9acd9b Alpha | Master
OS: Windows 10
CPU: AMD Ryzen 5 3600 6-Core Processor
GPU:AMD Radeon RX 5500 XT

Other users I was attempting to netplay with have encountered the same problem.

Unsure if i'm missing something basic as this is my first time using RPCN, apologies but I did try to find any other information I could before reporting an issue.

**Comment by DarkTree01** [comment]
[BLAZBLUE [BLUS30392].log.gz](https://github.com/user-attachments/files/17105509/BLAZBLUE.BLUS30392.log.gz)


Forgot to attach a log file, hopefully this helps. Mostly the last 10~ or so lines are the relevant ones as they're when the network connection begins.

**Comment by Ninetime** [comment]
The room creation issue should be resolved once https://github.com/RPCS3/rpcs3/pull/16677 is merged.

**Comment by Ninetime** [comment]
Yep, this is resolved as of https://github.com/RPCS3/rpcs3/pull/16677
